src/test/librbd/test_librbd.cc

```diff
@@ -4232,6 +4232,68 @@ interval_set<uint64_t> round_diff_interval(const interval_set<uint64_t>& diff,
   return rounded_diff;
 }
 
+TEST_F(TestLibRBD, TestSizeImageSnap)
```

Comment:
At least to me, the current name suggests that the issue is with snapshot sizes -- but it's really with the diff:
```suggestion
TEST_F(TestLibRBD, SnapDiff)
```

Reply:
+1 for making things more precise. In addition, I've updated the commit and PR to reflect that.